independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/AbstractGenerator.java

```diff
@@ -10,6 +10,8 @@
 abstract class AbstractGenerator {
 
     static final String DEFAULT_PACKAGE = Arc.class.getPackage().getName() + ".generator";
+    static final String UNDERSCORE = "_";
+    static final String SYNTHETIC_SUFFIX = "Synthetic";
```

Comment:
Smaller class names mean smaller JARs :wink: Maybe use the abbreviation `Syn`?

Reply:
Well, this is what I call a premature optimization :smile:. No seriously the readability is IMHO much more important in this case.